src/Amalgam/evaluablenode/EvaluableNodeManagement.cpp

```diff
@@ -13,13 +13,13 @@ Concurrency::ReadWriteMutex EvaluableNodeManager::memoryModificationMutex;
 #endif
 
 const double EvaluableNodeManager::allocExpansionFactor = 1.5;
-const ExecutionCycleCountCompactDelta EvaluableNodeManager::minCycleCountBetweenGarbageCollects = 150000;
-
-EvaluableNodeManager::EvaluableNodeManager()
-{
-	firstUnusedNodeIndex = 0;
-	executionCyclesSinceLastGarbageCollection = 0;
-}
+#ifdef MULTITHREAD_SUPPORT
+const ExecutionCycleCountCompactDelta EvaluableNodeManager::minCycleCountBetweenGarbageCollectsPerThread = 150000;
+#else
+//make the next value constant if no threads
+const
+#endif
+ExecutionCycleCountCompactDelta EvaluableNodeManager::minCycleCountBetweenGarbageCollects = 150000;
```

Comment:
I'm not sure I understand why this was split out. It looks like the "perThread" one is not used differently than non-perThread? And they are also the same value. Useful for future changes maybe?